mythril/laser/plugin/plugins/plugin_annotations.py

```diff
@@ -67,3 +86,30 @@ def __copy__(self):
         result = WSDependencyAnnotation()
         result.annotations_stack = copy(self.annotations_stack)
         return result
+
+    def check_merge_annotation(self, annotation: "WSDependencyAnnotation") -> bool:
+        if len(self.annotations_stack) != len(annotation.annotations_stack):
```

Comment:
```suggestion
        if len(self.annotations_stack) != len(annotation.annotations_stack):
            # We can only merge worldstate annotations that have seen an equal amount of transactions
            # since the beginning of symbolic execution
```